packages/v4/patternfly-docs/content/accessibility/card/card.md

```diff
@@ -3,21 +3,109 @@ id: Card
+| `onExpand` | `CardHeader` | Adds a toggle button with a caret icon to control whether a card is expanded or collapsed, and to visually indicate the card is expandable. <br/><br/> If an expandable card does not include a `CardTitle`, there should be some other visual context in the `CardHeader` to convey the content of the card to users that rely on sight, such as an image. |
+| `toggleButtonProps` | `CardHeader` | Sets props on the card's toggle. **Required** when a card is expandable. |
+| `component` | `CardTitle` | Sets the base element for the sub-component (`div` element by default). When one of the six heading elements (`h1` to `h6`) is passed in, you must ensure no heading levels are skipped within the surrounding context of the page. |
+
+### Expandable toggle props
+
+When a card has expandable content, `toggleButtonProps` must be passed into the `CardHeader` sub-component with an object as its value. This object should usually be passed in with a structure similar to the following:
+
+```noLive
+  toggleButtonProps=({
+    id: ‘toggleId’, 
+    ‘aria-label’: 'Toggle label', 
+    ‘aria-labelledby’: 'titleId toggleId', 
+    ‘aria-expanded’: this.state.isExpanded
+  )}
+```
+
+The `aria-label` property gives the toggle an accessible name for assistive technologies to announce to users, e.g. "Toggle label, button". 
+
+The `aria-labelledby` should include the `id` of a `CardTitle` sub-component if one exists or the `id` of the card itself, as well as the `id` of the toggle. This will combine the accessible name of both the `CardTitle` or `Card` and toggle, e.g. "Card title Toggle label, button". The order in which the ID's are passed in will determine this combined accessible name.
+
+The `aria-expanded` property must be set as the card's current expanded state. This will allow assistive technologies to announce to users whether the card is expanded or collapsed, e.g. "Toggle label Card title, collapsed, button".
+
+The `aria-label` and `id` attributes can be omitted from the `toggleButtonProps`, with the `aria-labelledby` attribute being given a value of only "titleId".
+
+## HTML/CSS customization
+
+Various HTML attributes and PatternFly classes can be used for more fine-tuned control over accessibility.
+
+| Attribute or class | Applied to | Reason | 
+|---|---|---|
+| `aria-label` or `aria-labelledby` | `.pf-c-card` | Adds an accessible name to the card. If the card has a `.pf-c-card__title`, the `aria-labelledby` attribute should be passed in with a value of the card title's `id` attribute. Otherwise the `aria-label` attribute should be passed in to act as a title for the card. |
+| `.pf-m-non-selectable-raised` | `.pf-c-card` | Adds styling to visually indicate that a selectable card is disabled. |
+| `.pf-m-selectable-raised` or `.pf-m-selectable` | `.pf-c-card` | Adds styling to visually indicate that a card is selectable. |
```

Comment:
I dont think these style related classes are necessary on this page.

Reply:
So for these, the main reason I included them was because without them it would make it difficult for users to understand that they can interact with the cards via pointer devices/which card is currently selected, and they're not styles that are automatically applied on hover. Though this could bring into question to what extent style-related classes should be mentioned on a11y docs (if at all).